### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by cranmer** [ihrke/IPythonQL#1]
I was just about to do this myself, when I found this. 
I am guessing it can use some updates with the changes going on in Jupyter and IPython.
Some of the notebooks I have work and some don't, but I haven't spent any time to debug yet.

**Comment by ihrke** [comment]
yeah, an update would be nice. Don't really have a lot of time for fixing it but PRs are welcome!
